src/test/BondingManagerForceChangeDelegateFix.sol

```diff
@@ -148,8 +86,6 @@ contract BondingManagerForceChangeDelegateFix is DSTest {
         bytes32[] memory proof;
         CHEATS.prank(delegator);
 
-        CHEATS.expectEmit(true, true, true, true);
-        emit Bond(delegate, address(0), delegator, 500000000000000000000, 500000000000000000000);
```

Comment:
This event should not be removed.

This test is to check for the case where L2Migrator is [allowed to change delegate of the delegator](https://github.com/livepeer/protocol/blob/confluence/contracts/bonding/BondingManager.sol#L505).

The addresses are selected such that a delegatorPool doesn't exist for the delegate and the tokens are directly bonded instead of getting claimed from the pool  [by calling bondFor function](https://github.com/livepeer/arbitrum-lpt-bridge/blob/main/contracts/L2/gateway/L2Migrator.sol#L183-L188) (mimics a situation where the orchestrator hasn't migrated yet) 

In case the tokens are claimed from the delegatorPool, the final assertion will still hold true but TransferBond event will be emitted instead of Bond since `bondForWithHint` function is not called.